packages/liveblocks-core/src/client.ts

```diff
@@ -546,7 +521,22 @@ export function createClient<U extends BaseUserMeta = DU>(
   );
   const baseUrl = getBaseUrl(clientOptions.baseUrl);
 
-  const authManager = createAuthManager(options);
+  const currentUserIdStore = createStore<string | undefined>(undefined);
```

Comment:
Minor nit, but while we're touching this, I think it might be nice not to return the entire store instance in `[kInternal]`, but only expose the "getter":

```tsx
{
  [kInternal]: {
    // currentUserIdStore,                     // ❌
    getCurrentUserId: currentUserIdStore.get,  // ✅
}
```

Reply:
@nvie Usually we also need the "susbcriber" in React land to build a reactive hook out of it

Reply:
Oh yeah. I think in this case we're not using that though?

Reply:
We are 👀 https://github.com/liveblocks/liveblocks/pull/2028/files#diff-bc0bfa8da3e750647522fb8b3960dffbe94655e1679cb8978f894bb2c2696254R5

Reply:
Oh, nevermind then! I said nothing. This morning, I only checked `react` I think, not `react-ui`. Carry on please! 🙈